backend/chart.go

```diff
@@ -188,30 +216,8 @@ func (backend *Backend) ChartData() (*Chart, error) {
 			return nil, err
 		}
 
-		addChartData := func(coinCode coin.Code, timeseries []accounts.TimeseriesEntry, chartEntries map[int64]ChartEntry) {
```

Comment:
Nice. I would also move `toSortedSlice` out. It really isn't a big deal - you see somehing "sorted", you'll know immediately what it does without even looking at its code. But moving `toSortedSlice` out makes it easier to read the rest of the important logic in this method.